### DISCUSSION TITLE
Request for Hobbyist/Academic Permissions

### DISCUSSION THREAD

**Discussion by franklin611** [transitland/transitland#440] [Q&A] [answered]
Hello!

I was wondering how the process for submitting a request for Hobbyist/Academic GTFS permission works? I sent an email request outlining the project I am working on a academic/school project that requires the historical feed data. 

Was wondering how long such a request would take? I applied and sent an email using my school's email address which is linked to my account.

Thank you!

**Comment by drewda** [comment, marked as answer]
Hello, our team usually reviews these requests once a week. Sometimes we're all busy and it can take longer. Thanks for your interest and your patience.
